sdk/keyvault/azure-security-keyvault-common/inc/azure/keyvault/common/internal/keyvault_pipeline.hpp

```diff
@@ -141,7 +141,7 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Common { n
      * @param path The path for the request.
      * @return A unique ptr to an Http raw response.
      */
-    std::unique_ptr<Azure::Core::Http::RawResponse> GetResponse(
+    std::unique_ptr<Azure::Core::Http::RawResponse> Send(
```

Comment:
In winhttp, we call this `SendRequestAndGetResponse` That self-documents what this is doing and also reduces confusing between it and `pipeline.Send()`

Reply:
As long as a function returning `unique_ptr` is not called `Get...()`, we are in a better place. This rename is in Victor's code, and it was his naming recommendation, so I leave it with that for now, later there can be further renames.